test/layernorm/test_layernorm_fp16.cpp

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#include "gtest/gtest.h"
+#include "test_layernorm_util.hpp"
+
+template <ck::index_t N>
+using I = ck::Number<N>;
+
+template <typename Tuple>
+class TestLayernormFP16 : public ck::TestLayernorm<Tuple>
+{
+};
+
+// clang-format off
+using KernelTypes = ::testing::Types<
+//  XDataType, GammaDataType, BetaDataType, AccDataType, YDataType, Rank, NumReduceDim, BlockSize, MThreadClusterSize, KThreadClusterSize, MThreadSliceSize, KThreadSliceSize, XYSrcVectorDim, XSrcVectorSize, , GammaSrcVectorSize, BetaSrcVectorSize, YDstVectorSize>
+    std::tuple<ck::half_t, ck::half_t, ck::half_t, float, ck::half_t, I<2>, I<1>, I<256>, I<8>, I<32>, I<1>, I<8>, I<1>, I<8>, I<8>, I<8>, I<8>>,
```

Comment:
Could you test the case where Rank=3? Because the immediate application to this kernel is batched gemm output. Similar softmax test case for your reference: https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/639147432b6922bd8e4051ba751e4e63dd4eb196/test/softmax/test_softmax_fp32.cpp#L17-L18

Reply:
It is also desirable that this standalone layernorm can support use cases where NumRedeuceDim>1 just like the above softmax test case. But not necessarily in the PR.